jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/impl/ExasolSqlDialectIT.java

```diff
@@ -133,26 +133,26 @@ public void testDataTypeSelect() throws SQLException {
 
     @Test
     public void testIdentifierCaseSensitivity() throws SQLException, FileNotFoundException {
-        ResultSet result = executeQuery("SELECT * FROM \"Table_Mixed_Case\"");
+        ResultSet result = executeQuery("SELECT * FROM " + VIRTUAL_SCHEMA_MIXED_CASE + ".\"Table_Mixed_Case\"");
         matchLastRow(result, 1L, 2L, 3L);
```

Comment:
Call all methods that can throw assertion errors `assert...`.

Reply:
done